nemo/core/classes/modelPT.py

```diff
@@ -431,9 +431,12 @@ def setup_multiple_test_data(self, test_data_config: Union[DictConfig, Dict]):
             if self._test_dl is not None and type(self._test_dl) in [list, tuple]:
                 self._test_names = ['test_{}_'.format(idx) for idx in range(len(self._test_dl))]
 
-    def setup_optimization(self, optim_config: Optional[Union[DictConfig, Dict]] = None):
-        """
-        Prepares an optimizer from a string name and its optional config parameters.
+    def setup_optimization(
+            self,
+            optim_config: Optional[Union[DictConfig, Dict]] = None,
+            optim_args: Dict[str, Any] = {},
```

Comment:
Should be optim_kwargs since it's a dict

Reply:
Also do not use {} in function, set default as None and check for not none before update below v